ikasaneip/platform/ikasan-setup/src/main/resources/db/data/db-changelog-removeRedundantSecurityRecords.xml

```diff
@@ -0,0 +1,190 @@
+        </delete>
+
+        <delete tableName="SecurityPolicy">
+            <where>Name = 'ViewWiretap'</where>
+        </delete>
+
+        <delete tableName="RolePolicy">
+            <where>PolicyId = (SELECT Id FROM SecurityPolicy WHERE Name = 'ViewExclusion')</where>
+        </delete>
+
+        <delete tableName="SecurityPolicy">
+            <where>Name = 'ViewExclusion'</where>
+        </delete>
+
+        <delete tableName="RolePolicy">
+            <where>PolicyId = (SELECT Id FROM SecurityPolicy WHERE Name = 'ViewActionedExclusions')</where>
+        </delete>
+
+        <delete tableName="SecurityPolicy">
+            <where>Name = 'ViewActionedExclusions'</where>
+        </delete>
+
+        <delete tableName="RolePolicy">
+            <where>PolicyId = (SELECT Id FROM SecurityPolicy WHERE Name = 'ViewErrors')</where>
+        </delete>
+
+        <delete tableName="SecurityPolicy">
+            <where>Name = 'ViewErrors'</where>
+        </delete>
+
+        <delete tableName="RolePolicy">
+            <where>PolicyId = (SELECT Id FROM SecurityPolicy WHERE Name = 'ViewCategorisedErrors')</where>
+        </delete>
+
+        <delete tableName="SecurityPolicy">
+            <where>Name = 'ViewCategorisedErrors'</where>
+        </delete>
+
+        <delete tableName="RolePolicy">
+            <where>PolicyId = (SELECT Id FROM SecurityPolicy WHERE Name = 'ActionExclusion')</where>
+        </delete>
+
+        <delete tableName="SecurityPolicy">
+            <where>Name = 'ActionExclusion'</where>
+        </delete>
+
+        <delete tableName="RolePolicy">
+            <where>PolicyId = (SELECT Id FROM SecurityPolicy WHERE Name = 'CreateBusinessStream')</where>
+        </delete>
+
+
+        <delete tableName="SecurityPolicy">
+            <where>Name = 'CreateBusinessStream'</where>
+        </delete>
+
+        <delete tableName="RolePolicy">
+            <where>PolicyId = (SELECT Id FROM SecurityPolicy WHERE Name = 'ModifyBusinessStream')</where>
+        </delete>
+
+        <delete tableName="SecurityPolicy">
+            <where>Name = 'ModifyBusinessStream'</where>
+        </delete>
+
+        <delete tableName="RolePolicy">
+            <where>PolicyId = (SELECT Id FROM SecurityPolicy WHERE Name = 'DeleteBusinessStream')</where>
+        </delete>
+
+        <delete tableName="SecurityPolicy">
+            <where>Name = 'DeleteBusinessStream'</where>
+        </delete>
+
+        <delete tableName="RolePolicy">
+            <where>PolicyId = (SELECT Id FROM SecurityPolicy WHERE Name = 'ViewTopology')</where>
+        </delete>
+
+        <delete tableName="SecurityPolicy">
+            <where>Name = 'ViewTopology'</where>
+        </delete>
+
+        <delete tableName="RolePolicy">
+            <where>PolicyId = (SELECT Id FROM SecurityPolicy WHERE Name = 'ViewMapping')</where>
+        </delete>
+
+        <delete tableName="SecurityPolicy">
+            <where>Name = 'ViewMapping'</where>
```

Comment:
How come we removing all these policies?

Reply:
This should have just removed the specific business stream not required for a vanilla deployment of Ikasan.

Reply:
These have all been replaced using the new policy scheme. I manually deleted them from our production when i released the new scheme so that i could manage without breaking. This is done for completeness. See db-changelog-policyRefactor.xml which contains all the policies that have superseded the ones deleted in this change.